ui/lib/ldap/addon/components/page/library/details/accounts.hbs

```diff
@@ -1,7 +1,7 @@
-<div class="has-top-margin-l is-flex-align-baseline">
-  <Hds::Card::Container @hasBorder={{true}} class="has-padding-l is-flex-half">
+<div class="has-top-margin-l is-flex-align-start">
+  <Hds::Card::Container @level="mid" @hasBorder={{true}} class="has-padding-l is-flex-half">
```

Comment:
Yes, only because the action in the toolbar for the overview card is a `LinkTo` and this calls for a button to trigger an action. Admittedly this is a cop out and the overview card could be updated to yield a block for the action which I think would be a better pattern than passing in args for the link but that would involve updating all of the existing instances. What do you think should I go for it in this PR or maybe do it as a follow up task or leave as is for now?

Reply:
Oh, I see. I think that would be a really great update to the overview card, but can probably be a follow-on from this PR